files/js/mapdata-skellige.js

```diff
@@ -1069,7 +1079,7 @@
 		}, {
 			coords: [[-54.098, -60.754]],
 			label: 'Wild Shore',
-			popup: 'A wild and untamed� part of the isle\'s coastline. A favourite spot for bandits and lovers'
+			popup: 'A wild and untamed™ part of the isle\'s coastline. A favourite spot for bandits and lovers'
```

Comment:
I'm not quite sure how the trademark symbol got added to this line. I surely don't remember doing.